sap-schemas/v2.0/README.md

```diff
@@ -381,6 +381,37 @@ Example:
 }
 ```
 
+### `x-sap-odm-semantic-key`
+
+- Type: `Array`
+- Used at: [Schema Object](https://spec.openapis.org/oas/v2.0#schema-object)
+- Description: Semantic key(s).
```

Comment:
I think we need more detailed description here. 1-2 sentences.

Reply:
Description has been changed.